libs/gisday/ngx/src/lib/pages/admin/pages/base-admin-detail/base-admin-detail.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Directive, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+
+import { Subject } from 'rxjs';
+import { debounceTime } from 'rxjs/operators';
+
+import { BaseService } from '@tamu-gisc/gisday/data-access';
+
+@Directive()
```

Comment:
https://stackoverflow.com/a/47883806
https://angular.io/guide/migration-undecorated-classes

Reply:
Same thing with the last one. I think semantically it makes more sense to have the component decorator instead of directive since they are two different things.